automated/linux/mmtests/json-to-lava.py

```diff
@@ -23,7 +24,13 @@ def main(args):
                 values = op_data["Values"]
                 samples = op_data["SampleNrs"]
                 for val, sample in zip(values, samples):
-                    print(f"{module_name}-{test_name}-{op}-{i}-{sample} pass {val}")
+                    print(f"{module_name};{op};{i};{sample} pass {val}")
```

Comment:
The "-" as a delimiter breaks the formatting, since some operations contain "-" in the name, the resulting string would be interpreted with one argument more, breaking the script that converts back to json

Reply:
@federico-gelmo can you add "_" instead of ";"

Reply:
Check the new patch.
The delimiter is now "_", and since that was used to differentiate additional fields like _Cmd, i changed the prefixed symbol for those attribute to "$".
You can check the commit message for an example and further clarification